examples/tutorial_client_eventfilter.c

```diff
@@ -0,0 +1,267 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
```

Comment:
Tutorial are the general examples. Since the event filter are more advances, please remove the "tutorial" and add it as normal client example.

Reply:
We have changed the names of both tutorials as you wished.